src/mmw/mmw/settings/layer_settings.py

```diff
@@ -233,6 +233,33 @@
         'opacity': 0.618,
         'has_opacity_slider': True,
         'perimeter': pa_perimeter,
+    },
+    {
+        'code': 'drb_catchment_water_quality_tn',
+        'display': 'DRB Catchment Water Quality Data TN Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tn',
+        'vector': True,
+        'overlay': True,
+        'minZoom': 3,
+        'perimeter': drb_perimeter
+    },
+    {
+        'code': 'drb_catchment_water_quality_tp',
+        'display': 'DRB Catchment Water Quality Data TP Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tp',
+        'vector': True,
+        'overlay': True,
+        'minZoom': 3,
+        'perimeter': drb_perimeter
+    },
+    {
+        'code': 'drb_catchment_water_quality_tss',
+        'display': 'DRB Catchment Water Quality Data TSS Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tss',
+        'vector': True,
```

Comment:
Not sure how people actually use these layers, but this looks really neat:

<img width="1197" alt="screen shot 2016-10-05 at 4 09 42 pm" src="https://cloud.githubusercontent.com/assets/7633670/19129786/8fac21d2-8b16-11e6-9032-6bb149b0ca02.png">